README.md

```diff
@@ -47,8 +47,35 @@ For a more detailed example, see `usage.py`.
 
 ### Quickstart (R)
 
-This section will be added soon.
+First, install the package from GitHub (the package is not yet available via CRAN):
+```
+remotes::install_github("plotly/dash-deck")
+```
+
+then, create your component and add it to your layout:
+
+```r
+library(dash)
+library(dashVtk)
+library(dashHtmlComponents)
```

Comment:
This is totally :ok_hand: for the moment, but once the unification PR is merged (https://github.com/plotly/dashR/pull/243), you'll just have to omit the third line, so instead users will load

```r
library(dash)
library(dashVtk)
```